ui/src/components/UserProfile/UserProfile.jsx

```diff
@@ -0,0 +1,105 @@
+import * as React from "react"
+import "./UserProfile.css"
+import axios from "axios"
+import * as config from '../../config'
+import * as Bootstrap from "react-bootstrap"
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+
+export default function UserProfile({ user }) {
```

Comment:
This file would benefit from refactoring. 
For eg - <li className="mb-2 mb-xl-3 display-28"><span className="display-26 text-secondary me-2 font-weight-600"> could be its own components and you can reuse it.

A lot of the css classnames are not very readable. 
For eg - card-body p-1-9 p-sm-2-3 p-md-6 p-lg-7
I understand they are for responsiveness. Just be careful as they will make your life more difficult in editing them later on.